src/lib/alloc.c

```diff
@@ -41,9 +41,8 @@
 
```

Comment:
Can you please describe what is the problem this change tries to fix?

Reply:
Other DEBUG_* features are already  settable from the config. This one was #define what we don't want. Just aligning to other CONFIG_DEBUG_* symbols.